libs/tracing/ppx_tests/trace_debug.ml

```diff
@@ -0,0 +1,17 @@
+let top a b =
+  let add_one a = a + 1 [@@trace] in
+  add_one a + b
+
+let top a b =
+  let add_one a = a + 1 [@@trace_debug] in
+  add_one a + b
+
+let top2 a b =
+  let%trace sp = "example" in
+  a + 1;
+  Tracing.add_data_to_span sp [ ("a", `Int 1) ]
+
+let top2 a b =
+  let%trace_debug sp = "example" in
```

Comment:
Yeah, I didn't implement it this way because I wasn't sure how to make it work with `let%trace`. I think it may be `let%trace sp = {name: "example"; level: "debug"}`; I'll try it out

Reply:
Left a comment